src/components/ui/ShippingSimulation/shipping-simulation.module.scss

```diff
@@ -6,54 +6,38 @@
   // --------------------------------------------------------
 
   // Default properties
-  --fs-shipping-simulation-text-size                : var(--fs-text-size-legend);
-
-  --fs-shipping-simulation-padding                  : var(--fs-spacing-5);
-
-  --fs-shipping-simulation-border-width             : var(--fs-border-width);
-  --fs-shipping-simulation-border-color             : var(--fs-border-color-light);
-  --fs-shipping-simulation-border-radius-bottom     : var(--fs-border-radius);
+  --fs-shipping-simulation-text-size: var(--fs-text-size-legend);
 
   // Title
-  --fs-shipping-simulation-title-padding-bottom     : var(--fs-spacing-2);
+  --fs-shipping-simulation-title-padding-bottom: var(--fs-spacing-2);
 
   // Link
-  --fs-shipping-simulation-link-padding-top         : var(--fs-spacing-1);
-  --fs-shipping-simulation-link-padding-bottom      : var(--fs-spacing-4);
+  --fs-shipping-simulation-link-padding-top: var(--fs-spacing-1);
+
+  // Header
+  --fs-shipping-simulation-header-padding-top: var(--fs-spacing-3);
 
   // Subtitle
-  --fs-shipping-simulation-subtitle-size            : var(--fs-text-size-2);
-  --fs-shipping-simulation-subtitle-weight          : var(--fs-text-weight-bold);
-  --fs-shipping-simulation-subtitle-line-height     : 1.5;
+  --fs-shipping-simulation-subtitle-size: var(--fs-text-size-2);
+  --fs-shipping-simulation-subtitle-weight: var(--fs-text-weight-bold);
+  --fs-shipping-simulation-subtitle-line-height: 1.5;
 
   // Location
-  --fs-shipping-simulation-location-padding-bottom  : var(--fs-spacing-2);
+  --fs-shipping-simulation-location-padding-bottom: var(--fs-spacing-2);
 
   // Table
-  --fs-shipping-simulation-table-row-bkg-color      : var(--fs-color-main-0);
-  --fs-shipping-simulation-table-row-even-bkg-color : transparent;
-
-  --fs-shipping-simulation-table-cell-padding       : var(--fs-spacing-2);
-  --fs-shipping-simulation-table-cell-text-size     : var(--fs-text-size-2);
-  --fs-shipping-simulation-table-cell-border-radius : .25rem;
+  --fs-shipping-simulation-table-row-bkg-color: var(--fs-color-main-0);
+  --fs-shipping-simulation-table-row-even-bkg-color: transparent;
```

Comment:
Could you please keep the same spacing for all tokens as before? Just for standards' sake